docs/examples/webhook.js

```diff
@@ -9,7 +9,7 @@ const Discord = require('discord.js');
 /*
  * Create a new webhook
  * The Webbooks ID and token can be found in the URL, when you request that URL, or in the response body.
- * https://discordapp.com/api/webhooks/12345678910/T0kEn0fw3Bh00K
+ * https://discord.com/api/webhooks/12345678910/T0kEn0fw3Bh00K
  *                                     ^^^^^^^^^^  ^^^^^^^^^^^^ 
  *                                     Webhook ID  Webhook Token
```

Comment:
Very tiny nit: those could be realigned to keep displaying correctly in examples.

Reply:
👍 Resolved